### ISSUE TITLE
"[Errno 97] Address family not supported by protocol" error when running in ubuntu 20.06 when ipv6 is disabled

### ISSUE THREAD

**Issue by rfeliciano129** [pyload/pyload#3849]
### Description

pyload 0.4.20 is failing to start the web-interface when run in ubuntu 20.06, if ipv6 is disabled. 
Same version of pyload runs fine on 14.04, also with ipv6 disabled. 

### Debug log

```
Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/rfeliciano/git/pyload/module/web/ServerThread.py", line 74, in run
    self.start_threaded()
  File "/home/rfeliciano/git/pyload/module/web/ServerThread.py", line 95, in start_threaded
    webinterface.run_threaded(host=self.host, port=self.port, cert=self.cert, key=self.key, cert_chain=self.cert_chain)
  File "/home/rfeliciano/git/pyload/module/web/webinterface.py", line 162, in run_threaded
    run(app=web, host=host, port=port, server=WSGI, quiet=True)
  File "/home/rfeliciano/git/pyload/module/lib/bottle.py", line 3137, in run
    server.run(app)
  File "/home/rfeliciano/git/pyload/module/web/utils.py", line 152, in run
    server.start()
  File "/home/rfeliciano/git/pyload/module/lib/wsgiserver/server.py", line 1525, in start
    raise socket.error(msg)
error: No socket could be created -- (('::', 7000, 0, 0): [Errno 97] Address family not supported by protocol)
```

**Comment by rfeliciano129** [comment]
When ipv6 is renabled, it seems to work fine. Python 2.7.18

**Comment by rfeliciano129** [comment]
Also, this is with the 0.0.0.0 default used for IP. If I give a real IP, it works. Looks like the default is to pass in a IPV6 style generic IP, but whatever library version comes on Ubuntu 20.04 is puking on it.

**Comment by treysis** [comment]
What if you set dualstack to "False"? In that case it should substitute `'::'` with `'0.0.0.0'`.

(and you shouldn't disable IPv6 😏)

**Comment by rfeliciano129** [comment]
That does work but I had to go into the yank to disable it. Figured the setup routine shouldn’t create an useable scenario
Also I’m nkt a fan of ipv6 at all. 
Thanks!


Sent from Yahoo Mail for iPhone


On Thursday, December 10, 2020, 1:31 AM, treysis <notifications@github.com> wrote:




What if you set dualstack to "False"? In that case it should substitute '::' with '0.0.0.0'.

(and you shouldn't disable IPv6 😏 )
